roles/tpa_single_node/templates/manifests/collector/osv/Deployment.yaml.j2

```diff
@@ -24,60 +25,71 @@ spec:
       annotations:
         configHash/auth: dac9d88322da8b671d9fc4f81500698ea05f108df8784b66db80d4242c9dee06
     spec:
+      volumes:
+        - name: guac-config
+          configMap:
+            name: collectorist-api
+        - name: tls
+          secret:
+            secretName: collector-osv-tls
+        - name: config-auth
+          configMap:
+            name: collector-osv-auth
       containers:
         - name: service
-          image: registry.redhat.io/rhtpa/rhtpa-trustification-service-rhel9@sha256:044b6070e52378a93cbcd65cbd5d10b866f102ee8d6040b92147df7b52202e83
+          image: "{{ tpa_single_node_trustification_image }}"
           imagePullPolicy: IfNotPresent
-          livenessProbe:
-            initialDelaySeconds: 2
-            httpGet:
-              path: /health/live
-              port: 9010
-          readinessProbe:
-            initialDelaySeconds: 2
-            httpGet:
-              path: /health/ready
-              port: 9010
-          command:
-            - /trust
+          command: ["/trust"]
           args:
             - collector
             - osv
             - -p
-            - '8080'
+            - "8080"
             - --auth-configuration
             - /etc/config/auth.yaml
             - --v11y-url
-            - https://v11y-api.trustificcation.svc.cluster.local
+            - https://v11y-api-pod:8080
             - --guac-url
-            - https://guac-graphql.trustificcation.svc.cluster.local/query
+            - https://guac-graphql-pod:8080/query
           env:
-            - name: NAMESPACE
-              valueFrom:
-                fieldRef:
-                  fieldPath: metadata.namespace
+#            - name: NAMESPACE
```

Comment:
I suggest to add above the following :
```suggestion
#  TODO
```
So we can keep track of those.

Reply:
Done